block/manager.go

```diff
@@ -437,6 +437,8 @@ func (m *Manager) BlockStoreRetrieveLoop(ctx context.Context) {
 			daHeight := atomic.LoadUint64(&m.daHeight)
 			for _, block := range blocks {
 				m.blockInCh <- newBlockEvent{block, daHeight}
+				m.logger.Debug("block retrieved from p2p block sync", "blockHeight", block.Height())
```

Comment:
nit: might as well add the daHeight too right? If we are logging one, it is helpful to log the other for reference.

Reply:
Added